election/models.py

```diff
@@ -8,6 +8,7 @@
 logger = logging.getLogger(__name__)
 
 MAX_NUM_GRADES = 20
+LANGUAGE_AVAILABLE = ['fr','en'] 
```

Comment:
Could you move both constants in the .env? I don't think it's a good idea to have constants lost in the files.